### ISSUE TITLE
Data explorer and nteract: Fix washed out projector rendering of grayscale

### ISSUE THREAD

**Issue by willingc** [nteract/archived-desktop-app#3159]
Large screen unable to properly display the details of light gray. Look at meeting accessibility guidelines for projection and colors.

Feedback from Elijah presentation (global-sprint).
